### ISSUE TITLE
map rmine2 crash on OpenGL 3.2 renderer

### ISSUE THREAD

**Issue by DX37** [yquake2/rogue#16]
So, when I playing rogue YamagiQ2 7.40 and going to `rmine2` (even `map rmine2`), it works fine on all renderers but OpenGL 3.2. It just kicks me out from the game with this:
![2019-05-31 12-25-38 Скриншот экрана](https://user-images.githubusercontent.com/12829428/58684860-ea943480-83a3-11e9-8d8c-ecbd11e5b356.png)

stderr.txt
```
Error: VirtualAlloc commit failed.
```

stdout.txt
```
]map rmine2
-------- game initialization -------
Game is starting up.
Game is rogue built on Feb  7 2019.
------------------------------------

------- server initialization ------
target_anger without target!
fixed TRIGGERED flag on trigger_once at (0 0 0)
52 entities inhibited.
0 teams repaired
0 teams with 0 entities.
monster_infantry in solid at (2360 -768 1037)
monster_soldier_light in solid at (3184 -152 1492)
------------------------------------

[0.0.0.0]:0: client_connect


-===================================-

 Mine Engineering
Map: rmine2
==== ShutdownGame ====
Shutting down curl.
Shutting down OpenAL.
Shutting down input.
recursive shutdown
```

**Comment by DanielGibson** [comment]
We have recently released 7.41, can you try if the issue still happens with it?
http://www.yamagi.org/quake2/

**Comment by DX37** [comment]
> 7.41

Well, upgrading worked just fine. Thanks.
